cla_backend/apps/reports/management/commands/mi_cb1_report.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+import logging
+from datetime import date, timedelta
+from django.core.management.base import BaseCommand
+from reports.tasks import ExportTask
+from core.models import get_web_user
+from django.views.decorators.csrf import csrf_exempt
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "This runs the MCCB1sSLA report"
+
+    def handle(self, *args, **options):
+        self.create_report()
+
+    @csrf_exempt
+    def create_report(self):
+        date_from = (date.today() - timedelta(days=7)).strftime("%Y-%m-%d")
+        date_to = date.today().strftime("%Y-%m-%d")
+
+        report_data = '{{"action": "Export", "date_from": "{0}", "date_to": "{1}"}}'.format(date_from, date_to)
+
+        web_user = get_web_user()
+        filename_of_report = "scheduled-mi-cb1-extract.csv"
```

Comment:
Does the report get overwritten if you run the report again or some additional processing is added to make the filename unique (i.e adding a date/time suffix)

Reply:
In ExportTaskBase() the filename is set with a date attached, so we do not over write reports.